lib/foreman_scap_client/client.rb

```diff
@@ -36,7 +36,18 @@ def scan_command
                                else
                                  ''
                                end
-      "oscap xccdf eval #{fetch_remote_resources} #{profile} #{tailoring_subcommand} --results-arf #{results_path} #{config[@policy_id][:content_path]}"
+      "oscap xccdf eval #{fetch_remote_resources} #{local_files_subcommand} #{profile} #{tailoring_subcommand} --results-arf #{results_path} #{config[@policy_id][:content_path]}"
+    end
+
+    def local_files_subcommand
+      supports_local_file_option? && !config[:fetch_remote_resources] ? '--local-files /root' : ''
+    end
+
+    def supports_local_file_option?
+      # OpenSCAP 1.3.6 and newer requires the `--local-files` option to use local copies of remote SDS components
+      version, _stderr, status = Open3.capture3('rpm', '-q', '--qf', '%{version}', 'openscap')
```

Comment:
There is no `rpm` in Debian-based distributions.
Why don't you use 
```bash
oscap --version | grep "OpenSCAP command line tool (oscap)" |  awk '{print $NF}'
```
instead of `rpm`?
Now the whole foreman_scap_client is broken in Debian-based distributions. https://github.com/theforeman/foreman_scap_client/issues/47

Reply:
You already filed an issue about this. Echoing the same thing in multiple places most likely will not help anything.